docs/admin/shipping-bytecode.md

```diff
@@ -0,0 +1,108 @@
+# EBPF Bytecode Image Specifications
+
+## Introduction
+
+The EBPF Bytecode Image specification defines how to package EBPF bytecode
+as container images. The initial primary use case focuses on the containerization
+and deployment of EBPF programs within container orchestration systems such as
+Kubernetes.  Where it is necessary to provide a portable way to distribute
+bytecode to all nodes which need it.
+
+## Specifications
+
+We provide two distinct spec variants here to ensure interoperatiblity with existing registries
+and packages which do no support the new custom media types defined here.
+
```

Comment:
Can you add a bullet list with the names of the two spec variants here.

Reply:
Done, this is a rough draft eventually this may turn into 2 separate documents like solo did for wasm modules [here](https://github.com/solo-io/wasm/blob/master/spec/README.md)